### ISSUE TITLE
Bad links in cheat sheet

### ISSUE THREAD

**Issue by StefRe** [rougier/scientific-visualization-book#18]
While the links are OK in the original cheat sheet https://github.com/rougier/scientific-visualization-book/blob/master/figures/cheatsheets/cheatsheets.pdf, they get corrupted in the individual sheet 1
 https://github.com/rougier/scientific-visualization-book/blob/master/figures/cheatsheets/cheatsheets-1.pdf and hence in the book:
![grafik](https://user-images.githubusercontent.com/19879328/142621466-31a89871-242e-4f27-9275-2603008315a0.png) 

(this may or may not be linked to [switch to Font Awesome for icons and brand logos](https://github.com/matplotlib/cheatsheets/commit/c631159844a8b0e602aa4274df84689bc475c956)).

**Comment by rougier** [comment]
Oh thanks. Could you make a PR (else I can correct myself)?

**Comment by StefRe** [comment]
I tried but I have only very basic knowledge of `tex` and didn't succeed. So I'm afraid I won't be of any help here.

**Comment by rougier** [comment]
Oh wait, you're right. I directly included the PDF cheatsheets and this would require to change the PDF inside the book repo. I'll let this issue open until next edition (since matplotlib cheatsheets are continuously modified).

**Comment by StefRe** [comment]
Hm, I'm not sure: as I wrote above, the big cheatsheet is fine, no need to change it for this reason. The issue comes from including the trimmed graphics:
https://github.com/rougier/scientific-visualization-book/blob/e131b436923190d1381d0711b24e1ab7af2741e5/figures/cheatsheets/cheatsheets-1.tex#L5

I read a bit about it and tried it with `adjustbox` as suggested on tex.stackexchange but with the same result (I'm completely new to tex).

**Comment by rougier** [comment]
Could you try the PDF version included in the pdf repository in this repo to check if you got the same problem?

**Comment by StefRe** [comment]
I *did* use the PDF version included in this repo, see links in the original post above.  

I also checked out matplotlib/cheatsheets@2e1d87bd3d03 which included png graphic icons instead of fontawsome icons and tried it with this pdf: same result (so I stroke through my assumption above).

**Comment by rougier** [comment]
For me, links seemed to be correct. Weird.

**Comment by StefRe** [comment]
Sorry, maybe I didn't express myself unambiguously: the links in the [complete cheat sheet](https://github.com/rougier/scientific-visualization-book/blob/master/figures/cheatsheets/cheatsheets.pdf) work, but after splitting them into [individual pages](https://github.com/rougier/scientific-visualization-book/blob/master/figures/cheatsheets/cheatsheets-1.pdf) and hence in the [book](https://github.com/rougier/scientific-visualization-book/blob/master/pdf/book.pdf) (page 236) they are corrupted (I change the title of this issue to make it clearer).   

Do the links **in the book** indeed work for you?

**Comment by rougier** [comment]
Yes, I was talking about book links. But I think I've read somewhere (stack overflow ?) that when you include PDF inside latex, links are no preserved and I wonder how I end up with working links.
